docs/conf.py

```diff
@@ -147,5 +147,26 @@
 
 # qiskit package specific variables
 html_context = {
-    'analytics_enabled': True
+    'repo_name': 'Example Repo',
+    'analytics_enabled': True,
+    'version_list': [0.1, 0.2, 0.3],
+    'sidebar_headings': [
```

Comment:
Do you think this is a better approach than adding an additional `toctree` in `indez.rst` beyond the main one like how qiskit meta is doing?

Reply:
the only way i could get the dropdowns to work with subpages is by specifying them here. If you just put it in the toctree in index.rst you don't get to customise what goes in the dropdown menu, it will just automatically populate it with the headings on the page and doesn't include sub pages